comment_spell_check/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""comment_spell_check"""
+
+from comment_spell_check import comment_spell_check
+
+
+def main():
```

Comment:
Defining main here seems odd to me. What is the purpose?

Reply:
Not sure.  I copied it from somewhere.

Reply:
It has to do with creating the "comment_spell_check" command line script when installing the package.

Reply:
Think that is configured to use another file and function here: https://github.com/SimpleITK/CommentSpellCheck/blob/main/pyproject.toml#L29